### PR TITLE
Roya internship

### PR DESCRIPTION
**Pull request by Royamosavipour**
Hi,
Some changes have been made in the project, please review and advise if possible, thanks
Br

### CODE REVIEW

File: app/src/components/Project/Content/AllCardsContent.tsx [hunk 1]
```diff
@@ -140,6 +161,149 @@ export default function AllCardsContent({
...
+        <div className='h-[800px] flex flex-col'>
+          <div onClick={toggleText} className='font-medium cursor-pointer m-3 text-xl font-bold'>
+            EROS
+          </div>
+          <div className={clsx(showTextEros ? 'block' : 'hidden', 'ml-3')}>
+            The Enterprise Resource Optimization System (EROS) is a cutting-edge IT project designed
+            to revolutionize the way our organization manages and utilizes its resources. EROS
+            integrates advanced technologies and smart algorithms to streamline and optimize key
+            business processes, ensuring maximum efficiency and resource utilization across
+            departments.{' '}
```
Comment (vlad-x) [inline app/src/components/Project/Content/AllCardsContent.tsx:210]: @Royamosavipour this text is supposed to come from project description, we don't need it hardcoded here like this.

@eugeek  FYI

File: app/src/components/Project/Content/AllCardsContent.tsx [hunk 2]
```diff
@@ -140,6 +161,149 @@ export default function AllCardsContent({
...
+          </div>
+          <div className='flex items-center gap-x-1.5 text-[#8279BD] font-semibold m-3'>
+            <div>Archived topics</div>
+            <ArrowSquareRight />
+          </div>
+
+          <div className='flex mt-4 justify-between  items-center border-y-2 border-border-color'>
+            <div className='font-bold ml-3 font-monaspace'>Invite link</div>
+            <div className='text-sm my-5'>https://dun.wtf/KfXvOA1e47oQy1ttnWrc</div>
```
Comment (vlad-x) [inline app/src/components/Project/Content/AllCardsContent.tsx:219]: @Royamosavipour same with this link here - it should have the project ID

**Review by vlad-x — CHANGES_REQUESTED** [review]
@Royamosavipour please check the comments

**Comment by eugeek** [comment]
I'll make rebase and fix it little bit later 🤔

File: app/src/components/Project/Content/AllCardsContent.tsx (hunk 1 above)
Comment (Royamosavipour) [reply app/src/components/Project/Content/AllCardsContent.tsx:210]: Thank you for your guidance

**Comment by eugeek** [comment]
Took changes with ProjectSettingsModal to new pull request

**Pull request closed without merging**